shared/constants/metametrics.ts

```diff
@@ -848,3 +848,18 @@ export enum MetaMetricsEventUiCustomization {
 export enum MetaMetricsContextProp {
   PageTitle = 'location',
 }
+
+/**
+ * The status on which to filter the returned regulations.
+ * Mentioned here: https://docs.segmentapis.com/tag/Deletion-and-Suppression#operation/listRegulationsFromSource
+ */
+export enum DeleteRegulationStatus {
+  failed = 'FAILED',
+  finished = 'FINISHED',
+  initialized = 'INITIALIZED',
+  invalid = 'INVALID',
+  notSupported = 'NOT_SUPPORTED',
+  partialSuccess = 'PARTIAL_SUCCESS',
+  running = 'RUNNING',
+  unknown = 'UNKNOWN',
+}
```

Comment:
nit: we usually use PascalCase for enum keys (e.g. `DeleteRegulationStatus.Failed` vs. `DeleteRegulationStatus.failed`).

```suggestion
export enum DeleteRegulationStatus {
  Failed = 'FAILED',
  Finished = 'FINISHED',
  Initialized = 'INITIALIZED',
  Invalid = 'INVALID',
  NotSupported = 'NOT_SUPPORTED',
  PartialSuccess = 'PARTIAL_SUCCESS',
  Running = 'RUNNING',
  Unknown = 'UNKNOWN',
}
```